<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>656</width>
    <height>474</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>641</width>
      <height>391</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>656</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actOpenDB"/>
   <addaction name="separator"/>
   <addaction name="actInsertRec"/>
   <addaction name="actEditRec"/>
   <addaction name="actDeleteRec"/>
   <addaction name="separator"/>
   <addaction name="actScan"/>
   <addaction name="actQuit"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actOpenDB">
   <property name="text">
    <string>打开数据库</string>
   </property>
   <property name="toolTip">
    <string>打开数据库</string>
   </property>
  </action>
  <action name="actInsertRec">
   <property name="text">
    <string>插入记录</string>
   </property>
   <property name="toolTip">
    <string>插入记录</string>
   </property>
  </action>
  <action name="actEditRec">
   <property name="text">
    <string>编辑记录</string>
   </property>
   <property name="toolTip">
    <string>编辑记录</string>
   </property>
  </action>
  <action name="actDeleteRec">
   <property name="text">
    <string>删除记录</string>
   </property>
  </action>
  <action name="actScan">
   <property name="text">
    <string>涨工资</string>
   </property>
   <property name="toolTip">
    <string>涨工资</string>
   </property>
  </action>
  <action name="actQuit">
   <property name="text">
    <string>退出</string>
   </property>
   <property name="toolTip">
    <string>退出</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>actQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>327</x>
     <y>236</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
